src/instructlab/config/init.py

```diff
@@ -333,6 +333,18 @@ def is_hpu_available() -> bool:
         return False
 
 
+def get_cuda_device_properties(gpu_num: int) -> tuple[str, int]:
```

Comment:
can we move this to utils? 
I plan to create a utils package and break up the file itself into smaller related groups
would be helpful to have everything in utils.py until then